app/containers/Forms/VerificationStatement.tsx

```diff
@@ -0,0 +1,178 @@
+import { VerificationStatement_application } from "__generated__/VerificationStatement_application.graphql";
+
+function formatBytes(bytes: number, decimals = 2) {
+  if (bytes <= 0) return "0 Bytes";
+
+  const k = 1024;
+  const dm = decimals < 0 ? 0 : decimals;
+  const sizes = ["Bytes", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"];
+
+  const i = Math.floor(Math.log(bytes) / Math.log(k));
+
+  return parseFloat((bytes / k ** i).toFixed(dm)) + " " + sizes[i];
+}
+
+interface Props {
+  application: VerificationStatement_application;
+  relay: RelayProp;
+  onError: () => void;
+}
+export const VerificationStatementComponent: React.FunctionComponent<Props> = ({
+  application,
+  relay,
+  onError,
+}) => {
+  const [isUploading, setIsUploading] = useState(false);
+  const saveAttachment = async (e) => {
+    setIsUploading(true);
+    const { environment } = relay;
+    const file = e.target.files[0];
+    const variables = {
+      connections: [application.attachmentsByApplicationId.__id],
+      input: {
+        attachment: {
+          file,
+          fileName: file.name,
+          fileSize: formatBytes(file.size),
+          fileType: file.type,
+          applicationId: application.rowId,
+          versionNumber: application.latestDraftRevision.versionNumber,
+        },
+      },
+    };
+
+    await createAttachmentMutation(environment, variables)
+      .catch(() => {
+        setIsUploading(false);
+      })
+      .then(() => setIsUploading(false));
+  };
+
+  return (
+    <div className="card">
+      <div className="card-header">
+        <Row>
+          <Col md={8}>
+            <h1>Verification Statement</h1>
+          </Col>
+        </Row>
+      </div>
+      <Row style={{ padding: "0 2em 2em 2em" }}>
+        <Col xs={12} style={{ margin: "20px 0 20px" }}>
+          {isUploading ? (
+            <div>
+              <div className="loadingSpinnerContainer">
+                <LoadingSpinner />
+                <span>Uploading file...</span>
+              </div>
+            </div>
+          ) : (
+            <div className="upload-button-container">
+              <FilePicker
+                onChange={saveAttachment}
+                onError={onError}
+                name="upload-attachment"
+              >
+                Upload New Attachment
+              </FilePicker>
+            </div>
+          )}
+        </Col>
+        <Col xs={12} style={{ margin: "20px 0 20px" }}>
+          {application.attachmentsByApplicationId.edges.map(({ node }) => {
+            return (
+              <>
+                <a href="#" className="attachment-link" key={node.id}>
+                  {node.fileName}
+                </a>{" "}
+                <FontAwesomeIcon
+                  icon={faTrash}
+                  onClick={() => {
+                    const { environment } = relay;
+                    return deleteAttachmentMutation(environment, {
```

Comment:
**To be done in a separate PR:** 
This deletes the attachment from the database, but not the GCS bucket. We will need to use https://www.npmjs.com/package/@google-cloud/storage to delete the file from GCS & then remove it from the database if the GCS delete was successful.